src/createHandlebars.ts

```diff
@@ -7,6 +7,27 @@ import { cases } from './change-case';
 // TODO: https://transcend.height.app/T-30349 - de-duplicate this with cli and node-util
 export { Handlebars };
 
+/**
+ * Given a file name, return the desirable name for a handlebars extension
+ * The extension is in camel case without the file suffix, and all slashes
+ * and spaces get removed
+ *
+ * @param fileName - Filename
+ * @param extension - File extension
+ * @returns Partial name
+ */
+export function getPartialNameFromFileName(
+  fileName: string,
+  extension = 'hbs',
+): string {
+  return cases.camelCase(
```

Comment:
Nitpick: You could use `path.extname(filename)` to find the extension and then use it in the function here